src/Aspire.Hosting.SqlServer/SqlServerContainerImageTags.cs

```diff
@@ -7,5 +7,6 @@ internal static class SqlServerContainerImageTags
 {
     public const string Registry = "mcr.microsoft.com";
     public const string Image = "mssql/server";
-    public const string Tag = "2022-latest";
+    // Tracking tag: 2022-latest. NOTE: latest  digest fails the tests. We need to investigate.
+    public const string Digest = "sha256:c4369c38385eba011c10906dc8892425831275bb035d5ce69656da8e29de50d8";
```

Comment:
Just curious - how did you get this?

Reply:
It wasn't straight forward as this is a past tag, and docker doesn't have a way of tracking that. It's easy to check the current digest of an image (tag) using `docker inspect`, but not the history. In this case, I checked the image supported tags at https://mcr.microsoft.com/en-us/product/mssql/server/tags, and saw that `2022-latest` points to the same one as `2022-CU14-ubuntu-22.04` and noticed there is also a `2022-CU13-ubuntu-22.04` which I assumed was the previous one and when running the tests it worked.